src/Raven.Client/ServerWide/Commands/GetNodeInfoCommand.cs

```diff
@@ -22,10 +23,35 @@ public class NodeInfo
         public RachisState CurrentState;
         public bool HasFixedPort;
         public int ServerSchemaVersion;
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue
+            {
+                [nameof(NodeTag)] = NodeTag,
+                [nameof(TopologyId)] = TopologyId,
+                [nameof(Certificate)] = Certificate,
+                [nameof(ClusterStatus)] = ClusterStatus,
+                [nameof(NumberOfCores)] = NumberOfCores,
+                [nameof(InstalledMemoryInGb)] = InstalledMemoryInGb,
+                [nameof(UsableMemoryInGb)] = UsableMemoryInGb,
+                [nameof(BuildInfo)] = BuildInfo,
+                [nameof(OsInfo)] = OsInfo,
+                [nameof(ServerId)] = ServerId.ToString(),
+                [nameof(CurrentState)] = CurrentState,
+                [nameof(HasFixedPort)] = HasFixedPort,
+                [nameof(ServerSchemaVersion)] = ServerSchemaVersion,
+            };
+        }
     }
 
     public class GetNodeInfoCommand : RavenCommand<NodeInfo>
     {
+        public GetNodeInfoCommand(TimeSpan? timeout = null)
```

Comment:
For backward binary compatibility, since this is in the Client, you need to have empty ctor also

Reply:
Done